frontend/src/components/CognitoLoginAndReg.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import { useLocation } from 'react-router-dom'
+
+function useQuery() {
+  return new URLSearchParams(useLocation().search);
+}
+
+export default () => {
+  const [, setTokenObj ] = useState({});
+  const [, setUserObj ] = useState({});
```

Comment:
I don't understand why we need state that we don't read. What would happen if you just got rid of it?